templates/partials/messages_content.html

```diff
@@ -57,7 +57,7 @@
           </ul>
         </div>
       </div>
-      <div id="message-footer" class="reply message-form" ng-if="selected.messages[0]">
+      <div id="message-footer" class="reply message-form" ng-if="selected.messages[0] && (selected.contact.name || selected.contact.doc)">
```

Comment:
Good call. The other way would be to keep the footer but disable the textarea but this is good for now.